README.md

```diff
@@ -4,27 +4,47 @@ https://b4mad.racing - https://pitwall.b4mad.racing
 
 ### A telemetry data collection pipeline for simracers :racing_car:
 
-The current implementation allows you to collect telemetry data from multiple racing games via a [SimHub](https://www.simhubdash.com/) Plugin, which will send the data to a server in the cloud, where it's stored in a time-series database. You can then visualize the data in a customizable dashboarding tool.
+The b4mad client is implemented via a [SimHub](https://www.simhubdash.com/) plugin. This means that b4mad can take data
```

Comment:
At some time we need to decide on the naming. http://b4mad.net is the home of a group of nerds :) Maybe we call the product `B4mad.racing` ? And the Cloud is the `B4mad.racing platform` ? But 🤷  for now.